traitlets/config/configurable.py

```diff
@@ -203,8 +210,13 @@ def _config_changed(self, change):
         section_names = self.section_names()
         self._load_config(change.new, traits=traits, section_names=section_names)
 
-    def update_config(self, config):
-        """Update config and load the new values"""
+    def update_config(self, config, skip_env=False):
```

Comment:
> ... ambigous as to whether it means "skip loading environment variables" or "skip config also set by environment variables".

Sorry, I think i may have missunderstood what you meant.  
Can you elaborate a bit?